src/sentry/tasks/post_process.py

```diff
@@ -1561,12 +1561,21 @@ def check_if_flags_sent(job: PostProcessJob) -> None:
 
 
 def kick_off_seer_automation(job: PostProcessJob) -> None:
+    from sentry.seer.issue_summary import get_issue_summary_cache_key
     from sentry.seer.seer_setup import get_seer_org_acknowledgement
     from sentry.tasks.autofix import start_seer_automation
 
     event = job["event"]
     group = event.group
 
+    # Only run on new issues or issues with no existing summary/scan
+    if (
+        not job["group_state"]["is_new"]
+        and group.seer_fixability_score is not None
```

Comment:
should these be `or`s not `and`s?

Reply:
They should be `and` because we want to scan any issue that is new, does not have a fixability score, or does not have a recent summary